src/cmd/team_test.go

```diff
@@ -0,0 +1,106 @@
+package cmd_test
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/opslevel/cli/cmd"
+	"github.com/opslevel/opslevel-go/v2024"
+)
+
+const (
+	teamFileName = "test_team.yaml"
+	teamName     = "CLI Test Team"
+)
+
+func Test_TeamCRUD(t *testing.T) {
+	// Create Team
+	teamToCreate := opslevel.TeamCreateInput{
+		Name:             teamName,
+		Responsibilities: opslevel.RefOf("all the things"),
+	}
+	teamId, err := createTeam(teamToCreate)
+	if err != nil {
+		t.Error(err)
```

Comment:
Request that we replace these `t.Error(err)` and `t.FailNow()` calls with `t.Fatal(err)`

https://pkg.go.dev/testing#T.FailNow

>Error is equivalent to Log followed by Fail.

>Fatal is equivalent to Log followed by FailNow.

Therefore `t.Fatal(err)` is the better call to make.